### DISCUSSION TITLE
Define columns shown on document editables?

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#12904] [Q&A / Support] [answered]
Is it possible to adjust the columns shown by default on [an editable like this](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Relations_(Many-To-Many).html)?

``` twig
            {{ pimcore_relations('business_unit', {
                title: "Business unit (Category)",
                types: ['object'],
                classes: ['Category'],
                height: 150,
                width: 500,
                disableInlineUpload: true,
                reload: true
            }) }}
```

![bild](https://user-images.githubusercontent.com/279826/185094726-ca56e805-1627-4a6a-8ede-45b333244335.png)

**Comment by ChrisB9** [comment, marked as answer]
No its not possible currently. All the column entries are hardwired here: https://github.com/pimcore/pimcore/blob/11.x/bundles/AdminBundle/Resources/public/js/pimcore/document/editables/relations.js
I solved it for one of my use-cases in a way that I created a custom editable extending this and adding my own hard-wired columns
